<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>12</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Device Index:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_deviceIndex"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_scanDevice">
              <property name="text">
               <string>Scan Device</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_openDevice">
              <property name="text">
               <string>Open Device</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Format:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_format">
              <item>
               <property name="text">
                <string>standard</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>extended</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_type">
              <item>
               <property name="text">
                <string>data frame</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>remote frame</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Mode:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_mode">
              <item>
               <property name="text">
                <string>normal</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>loopback</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>listen-only</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>BaudRate:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_baudrate">
              <item>
               <property name="text">
                <string>20K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>33.33K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>40K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>66.66K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>80K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>83.33K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>125K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>200K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>250K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>400K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>500K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>666K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>800K</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1000K</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Sample Point:87%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Frame ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_frameId">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Data:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_data"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>NumberToSend(1-5000k):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_numToSend">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Send Interval(1ms-5000ms):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_sendInterval">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_IDAutoInc">
              <property name="text">
               <string>ID Auto Inc.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_DataAutoInc">
              <property name="text">
               <string>Data Auto Inc.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_hideErrorFrame">
              <property name="text">
               <string>Hide Error Frame:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton_delaysend">
              <property name="text">
               <string>Delayed Send</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_send">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_clear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_export">
              <property name="text">
               <string>Export</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>12</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>12</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
